common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2903,6 +2923,11 @@
       "3. set \"spark.sql.legacy.allowUntypedScalaUDF\" to \"true\" and use this API with caution."
     ]
   },
+  "UPDATE_FIELD_WITH_STRUCT_UNSUPPORTED" : {
```

Comment:
A sub-class of `UNSUPPORTED_FEATURE`?

Reply:
I want create a parent-class `INVALID_UPDATE_FIELD` in future.
There are a lot of similar error msg.

Reply:
SGTM